pkg/api/v1alpha1/nutanixmachineconfig.go

```diff
@@ -0,0 +1,104 @@
+package v1alpha1
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	NutanixMachineConfigKind        = "NutanixMachineConfig"
+	DefaultNutanixOSFamily          = Ubuntu
+	DefaultNutanixSystemDiskSizeGi  = "40Gi"
+	DefaultNutanixMemorySizeGi      = "4Gi"
+	DefaultNutanixVCPUsPerSocket    = 1
+	DefaultNutanixVCPUSockets       = 2
+	DefaultNutanixMachineConfigUser = "nutanix-user"
```

Comment:
Do we expect to use these defaults outside of the `NewNutanixMachineConfigGenerate` method?
If not we could make them non exported.

Reply:
Good catch; unexported them. Can always make it exported later if need arises.